### ISSUE TITLE
Add to calendar functionality update

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#1816]

**Comment by meols** [comment]
Add to calendar should have the additional functionality to have the date read by screenreader – like “Add <date> to calendar” “Add <title> to calendar”. “Add <date/title> to calendar”. Please suggest the best way to solve it.
Issue flagged in Siteimprove that we have Links in the same context with the same text alternative. https://my2.siteimprove.com/Accessibility/550893/NextGen/Issue/1?ruleId=81&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTarget.conformanceLevels=0,1,3

**Comment by padms** [comment]
@meols  Added a textsnippet under "Events" for the aria label which is used on the button.

**Comment by padms** [comment]
@meols  This is applicable to 'Add to calendar' button on the event page and event promotion cards. When tabbing through this button, screen reader reads the button as 'Add [event title] to calendar' .
 Sample links: 
https://web-global-development-equinor-web-sites-dev.c2.radix.equinor.com/test/padms
https://web-global-development-equinor-web-sites-dev.c2.radix.equinor.com/event/q3-2022

aria label is added to the button 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBaEZwIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--01510fe30ba24ffbf4f7fca36b6208bd1db3b3a0/image.png)

**Comment by padms** [comment]
@meols Removed quotes from the label

**Comment by meols** [comment]
@padms - shouldn't the marked code read _Add This is an event with 60 characters to calendar_?
If {eventTitle} is read out load by a screen reader, it doesn't identify the event.
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdUxnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--9427b0bfe2cca0080118d443b3edfc3d6707af0b/image.png)

**Comment by padms** [comment]
@meols We just need to update the text snippet. It shows correct text now.

**Comment by padms** [comment]
@meols Yes, This is how it looks after removing quotes from the text snippet. [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbHZvIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--c13873eec0f6ccc1de9266d40e7897a6b9cbce5e/image.png)
Can be testes on the same page now..

**Comment by meols** [comment]
Thanks @padms - not sure how to test this, but I trust you - so this is approved :smile:
